modules/untar/main.nf

```diff
@@ -2,10 +2,10 @@ process UNTAR {
     tag "$archive"
     label 'process_low'
 
-    conda (params.enable_conda ? "conda-forge::tar=1.34" : null)
+    conda (params.enable_conda ? "conda-forge::sed=4.7" : null)
```

Comment:
That's the only change that I don't get. Isn't it much more explicit to install tar in the untar module rather than sed which happens to include tar?

Reply:
Good spot! Missed that in the copy/paste. Wondering whether we need to care about this specifically because these tools should all be installed in the base Conda env anyway right? So do we go for standardisation here where we always define `sed` for modules using the `ubuntu` image or do we define the specific tools we are using? The latter could also be a pain in general because in most instances you will want to use multiple utilities like `cat`, `awk`, `sed`  in the same module etc

Reply:
Fair point. If there's a way to pull in multiple standard tools that are also available in the container images, then that's probably preferable. Would that make available `ps` btw? I think that's needed for resource tracing, right?